src/rgw/rgw_rest_s3.cc

```diff
@@ -2345,11 +2345,12 @@ static void dump_bucket_metadata(req_state *s, rgw::sal::Bucket* bucket)
   // only bucket's owner is allowed to get the quota settings of the account
   if (bucket->is_owner(s->user.get())) {
     auto user_info = s->user->get_info();
+    auto bucket_info = s->bucket->get_info().quota; // bucket quota
```

Comment:
This is the bucket quota, not the bucket info.  Either it should be named `bucket_quota` or it should store the info, like `user_info` does.

Reply:
Done!